src/autocomplete.js

```diff
@@ -274,16 +291,16 @@ class Autocomplete extends Component {
       () => {
         // call the provided callback if it's a callback
         cbToCb(cb)()
-        // if the selectedValue changed
```

Comment:
What if... we got rid of `onChange` and just pointed people to something like `onStateChange({currentState, prevState})`.